cassandra/1.cql

```diff
@@ -12,7 +12,8 @@ CREATE TABLE devgym.task (
   year TIMESTAMP,
   type TEXT,
   timeuuid TIMEUUID,
-  task_description TEXT,
+  name TEXT,
```

Comment:
But I think It is hard sometimes, we can use description whatever, it's not a problem :) Just my opinion, thanks for listening